test/integration/sharp/sharp.test.ts

```diff
@@ -0,0 +1,50 @@
+import { describe, expect, it } from "bun:test";
+import sharp from "sharp";
+
+describe("sharp integration tests", () => {
+  it("should resize an image", async () => {
+    const inputBuffer = await sharp("bun.png").resize(200, 200).toBuffer();
```

Comment:
```suggestion
    const inputBuffer = await sharp(path.join(import.meta.dir, "bun.png")).resize(200, 200).toBuffer();
```
also for the other uses of `bun.png`